web/styles/zulip.css

```diff
@@ -44,6 +44,11 @@ body {
         --header-height: 30px;
     }
 
+    --navbar-alerts-wrapper-height: 0px;
```

Comment:
It's probably useful to add a comment here noting that the actual value is set in `navbar_alerts.js`.

Reply:
Done. thanks!